src/AltinnCore/Testdata/User/2/partylist.json

```diff
@@ -3,9 +3,9 @@
 		"partyTypeName": "Person",
     "orgNumber": null,
     "person":  null,
-		"organization": null,
+		"organisation": null,
 		"partyID": "50002",
-		"organizationNumber": null,
+		"organisationNumber": null,
```

Comment:
is this variable ever used? orgNumber is what is defined in the party.cs..

Reply:
Don't know, I just changed it since the names should be the same.

Reply:
Not sure if we actually use this, since the orgNumber will be populated inside of the `organisation`-attribute on this object.